config.py

```diff
@@ -0,0 +1,29 @@
+import os
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config(object):
+    DEBUG = False
+    TESTING = False
+    CSRF_ENABLED = True
+    SECRET_KEY = 'this-really-needs-to-be-changed'
+    # SQLALCHEMY_DATABASE_URI = os.environ.get("DATABASE_URL")
```

Comment:
comments in the bin please (unless they're really comments that explain something useful in the codebase)